backend/src/create_meeting.rs

```diff
@@ -6,12 +6,45 @@ fn create_meeting(
     host: ft_sdk::Host,
     config: crate::Config,
     app_url: ft_sdk::AppUrl,
+    lets_auth_app_url: ft_sdk::AppUrl<"lets-auth">,
     scheme: crate::HTTPSScheme,
 ) -> ft_sdk::form::Result {
-    if !user.is_special(&config) {
-        return Err(title
-            .error("You are not authorized to create a meeting")
-            .into());
+    if let Err(e) = user.is_special(&config, &host) {
+        use crate::auth::AuthorizationError;
+        let msg = match e {
+            AuthorizationError::Unauthorized => {
+                "You are not authorized to create a meeting. Please contact the admin.".to_string()
+            }
+            AuthorizationError::EmptyWhitelist => {
+                "`who-can-create-meetings` variable is empty. You can't create a meeting."
+                    .to_string()
+            }
+            AuthorizationError::RequiresVerification => {
+                let verification_link = lets_auth_app_url.join(
+                    &scheme,
+                    &host,
+                    format!(
+                        "/{auth_wasm_name}/resend-confirmation-email/?email={email}",
```

Comment:
Not fond of `{auth_wasm_name}` here. Lets rename things to `backend.wasm` soon in auth (or create a wasm+proxy entry for it.

Reply:
Also since the user is logged in, the email address should not be required, can we make it optional for `resend-confirmation-email` API?

Reply:
A simple url-mapping did not work. I've created an issue (see https://github.com/fastn-stack/fastn/issues/2098). I'll rename `email_auth_provider` to `backend` for now.